oneanddone/base/templatetags/jinja_helpers.py

```diff
@@ -0,0 +1,116 @@
+import datetime
```

Comment:
I would recommend moving away from `|fe` and using jinja2's newstyle gettext and `|safe`, which IMO is much clearer in code and a more vanilla django/jinja2 way to go.

@willkg explains it way better in his bug https://bugzilla.mozilla.org/show_bug.cgi?id=1222040